test/src/examples_complete_test.go

```diff
@@ -39,43 +38,9 @@ func TestExamplesComplete(t *testing.T) {
 
 	// Run `terraform output` to get the value of an output variable
 	id := terraform.Output(t, terraformOptions, "id")
```

Comment:
the `id` of what resource?

Reply:
576c6bfc8b30c25a45d0c934d7477b2852d4dc17